### ISSUE TITLE
Some tests are gtk2 only

### ISSUE THREAD

**Issue by HinTak** [flatmax/gtkiostream#9]
ScrollingTest for example. I have both gtk2 and gtk3, and found that I needed to change $EXTRA_LIBS and $EXTRA_CFLAGS from GTK_* to GTK2_* in test to let it finish building.

**Comment by HinTak** [comment]
I have a local fix as outlined above, but I assume the correct change is possibly trying to make the test work with both, instead of limiting to gtk2

**Comment by flatmax** [comment]
I think this bug is most likely showing itself due to Distro upgrades defaulting to gtk3 ... its taken them a while.
Yep agreed, the headers have to be altered for specific classes to update them to be compat. with gtk3.
Another fix would be to put versioning #ifdef s in the effected files and in the test Makefile.am.

**Comment by HinTak** [comment]
Fedora has been gtk3 for quite a few years; Ubuntu is largely gtk3 for a couple of years now. You actually had one place in code , "include/Selection.H" where you check and error on gtk3. At least any file including that is meant to be gtk2 only. In the end I did the quick fix, which is changing "test/Makefile.am" around line 47, just after JackPortMonitorGuiTest/TimeTest to do GTK2 - which includes ScrollingTest - but maybe some of those work with gtk3 too.

**Comment by flatmax** [comment]
Yes, a large amount of the code works on gtk3. A few of the GUI 
components need updating.
